src/components/course/CourseRunCard.jsx

```diff
@@ -62,6 +64,9 @@ const CourseRunCard = ({
     subsidyRequestConfiguration,
   } = useContext(SubsidyRequestsContext);
 
+  const userHasSubsidyRequestForCourse = !!(subsidyRequestConfiguration?.subsidyRequestsEnabled)
+    && useUserHasSubsidyRequestForCourse(courseKey);
```

Comment:
[curious] I'm a bit surprised that React isn't complaining here about using conditional hooks? I believe it could be an issue to call the hook conditionally, per the React docs: 

https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

<img width="734" alt="image" src="https://user-images.githubusercontent.com/2828721/160189411-e1b66ff9-a1e1-4e7d-8198-479178cbc079.png">

Reply:
🤦